completion_aggregator/settings/common.py

```diff
@@ -13,18 +13,18 @@ def plugin_settings(settings):
     # This setting controls which type of event will be published to change the default behavior
     # the block type should be removed or added from the progress or completion list.
     settings.ALLOWED_COMPLETION_AGGREGATOR_EVENT_TYPES = {
-        "progress": [
+        "progress": {
```

Comment:
Why do you change from list to set ?

Reply:
https://github.com/open-craft/openedx-completion-aggregator/pull/173#discussion_r1415800773

Reply:
https://github.com/open-craft/openedx-completion-aggregator/pull/173#discussion_r1415800773